src/components/Admin.js

```diff
@@ -54,10 +51,6 @@ const Admin = () =>
                         path="/admin/users"
                         exact
                         component={Users}/>
-                    <Route
```

Comment:
I see what you're doing with using UserDetails for the stateless and User for the container, but do you guys need to keep the UserDetails page as it is for now? It looks like you're just getting rid of it altogether. If you wanna keep it I would move the stateless stuff into third file and leave UserDetails how it was.

Reply:
+1 to Tyler's comment

Reply:
I'm not 100% sure what you're saying - can you clarify?

Reply:
Essentially your changes are taking two pages: UserDetails and Users and combining them down to one page. Thats because you've converted UserDetails from being a standalone component to a stateless component that you're housing on the User page. I'm saying leave the UserDetails page alone so you still have both the UserDetails and the Users page, and then put all the stateless work into a third file. You can then call that stateless component from the Users page.

Reply:
This still hasn't been resolved. UserDetails and Users are meant to be two separate pages and you've basically made users a container for UserDetails. Please address